MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.m

```diff
@@ -60,33 +59,8 @@ - (id)initWithBaseUrl:(NSString *)baseUrl
   return self;
 }
 
-- (void)sendAsync:(MSLogContainer *)container completionHandler:(MSSendAsyncCompletionHandler)handler {
-  @synchronized (self) {
-    NSString *batchId = container.batchId;
-
-    // Verify container.
-    if (!container || ![container isValid]) {
-      NSDictionary *userInfo = @{NSLocalizedDescriptionKey: kMSMCLogInvalidContainerErrorDesc};
-      NSError *error =
-              [NSError errorWithDomain:kMSMCErrorDomain code:kMSMCLogInvalidContainerErrorCode userInfo:userInfo];
-      MSLogError([MSMobileCenter logTag], @"%@", [error localizedDescription]);
-      handler(batchId, error, nil);
-      return;
-    }
-
-    // Check if call has already been created(retry scenario).
-    id <MSSenderCall> call = self.pendingCalls[batchId];
-    if (call == nil) {
-      call = [[MSRetriableCall alloc] initWithRetryIntervals:_callsRetryIntervals];
-      call.delegate = self;
-      call.logContainer = container;
-      call.completionHandler = handler;
-
-      // Store call in calls array.
-      self.pendingCalls[batchId] = call;
-    }
-    [self sendCallAsync:call];
-  }
+- (void)sendAsync:(NSObject *)data completionHandler:(MSSendAsyncCompletionHandler)handler {
+  [self sendAsync:data callId:MS_UUID_STRING completionHandler:handler];
```

Comment:
Oh.. Yes. But it is still required because distribution http sender will have retry rules we have to maintain `MSSenderCall`.